### DISCUSSION TITLE
FIPS 140-2 Compliance?

### DISCUSSION THREAD

**Discussion by ghost** [argoproj/argo-cd#8834] [Q&A]
I am currently assembling compliance information for deployment tools and was unable to find any information on FIPS 140-2 compliance for cryptographic modules uses in ArgoCD. Can someone please point me in the right direction?

**Comment by alexmt** [comment]
I'm not sure if it will help, but you might try to check generated SBOM file attached to the release. E.g. https://github.com/argoproj/argo-cd/releases/download/v2.3.2/sbom.tar.gz . The tar has `bom-go-mod.spdx` file that lists all used go modules.

**Reply by ghost** [reply]
Thank you for that.

**Comment by faelin** [comment]
@tappl if you don't mind, could you share what you discovered?

**Comment by seanorama** [comment]
Also very interested in what you found. It may be enough to compile with BoringCrypto.

**Comment by bjagasia** [comment]
Very interested as well.
